Automattic/wp-calypso
# Replace internal PopupMonitor library with external

#### Changes proposed in this Pull Request

* Replace the internal `PopupMonitor` library with the external one published after #40972.

Note: the CSS change was to sort out a CSS class namespace lint error. That class was only used in this particular case, so there should be no regressions.

#### Testing instructions

* Head to `/marketing/connections/`.
* Try connecting to any of the available services.
* Make sure the popup show up as expected.
* Complete the authorization, and make sure the popup closes and the connection is established.
